### DISCUSSION TITLE
NodeGraph graph more flat

### DISCUSSION THREAD

**Discussion by eravion** [grafana/grafana#49028] [Dashboards, Panels & Visualizations]
Hello,

Here we have the doc about how it will look like :
https://github.com/grafana/grafana/blob/277a83215c1e20ff8c93768f93edf92d60deee5f/public/app/plugins/panel/nodeGraph/layout.worker.js#L56

Could it be posible to have something more flat
 0 - 0 - 0 - 0 - 0 - 0 - 0 - 0 - 0 - 0
. . . . . . .\ - 0 - /
 
Or an option to choose the look and feel ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
